manifests/setup/prometheus-operator-service.yaml

```diff
@@ -10,9 +10,9 @@ metadata:
 spec:
   clusterIP: None
   ports:
-  - name: http
-    port: 8080
-    targetPort: http
+  - name: https
+    port: 8443
```

Comment:
Whats the reason behind port change? Is it to signal end users that protocol changed?

Reply:
> Is it to signal end users that protocol changed?

Exactly for this reason. I am just adhering to best-practices :)